### ISSUE TITLE
[request] Remove 4k SDR quality from torrentio real debrid

### ISSUE THREAD

**Issue by shanestonk** [TheBeastLT/torrentio-scraper#283]
Is it possible to make torrentio just shown result of 4k hdr, hdr10+ and 4k dolby vision but it remove all the 4K SDR result (4k quality with non hdr).

i'd disable the 4k option on torrentio option but it removed all the 4k result and show only 1080p result eventhough i enable hdr, remux,dolby vision etc.

**Comment by catfluoride** [comment]
I disagree with this. Not every 4K TV supports HDR adequately (low nits, etc...), while 4K SDR is perfectly fine.

**Comment by Arcitec** [comment]
This is a bad suggestion even for yourself. Not all movies come in HDR versions so you would be removing all the sources for many of your movies.

**Comment by fairtale5** [comment]
> This is a bad suggestion even for yourself. Not all movies come in HDR versions so you would be removing all the sources for many of your movies.

that's not what he said?

**Comment by fairtale5** [comment]
> > that's not what he said?
> 
> That is EXACTLY what he said:
> 
> * "Delete ALL 4K RESULTS if they are not HDR".
> * Thereby, deleting all 4K movies that don't exist as HDR.

It shouldn't be hard to use a bit of imagination to think "this is a great request, let us be reasonable and implement it as an optional filter that each user can turn on or off. and even more: it could be just a SORT option, to show those results on top, and even more: this could be used not only to sort certain qualities, but also certain codex (like h265), or audio channels or even languages.

I honestly dont see your point but hey: at least you're participating.

**Comment by Arcitec** [comment]
@fairtale5 You have incredibly powerful reading comphrehension issues, and you are inserting yourself pointlessly with stupid comments.
